### PR TITLE
DEV-27807 Allow correct sorting by license status if the user has the license type Named user

### PR DESCRIPTION
**Pull request by mrnyax**
The user api needs to provide the possibility to sort the license type column also by status

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1331,6 +1331,7 @@ You can add a `+` or `-` to display the list in ascending or descending order re
 - Query parameter can't be empty or filled with blank spaces.
 - Query parameter can accept a mix of all characters.
 - By default, the list appears in ascending order.
+- Query parameter `licenseType` sorts by two fields `licenseType` and `licenseStatus`
```
Comment (tkrisztian95) [inline apiary.apib:1334]: I think a dot is missing from the end of this sentence:
```suggestion
- Query parameter `licenseType` sorts by two fields `licenseType` and `licenseStatus`.
```

File: apiary.apib [hunk 2]
```diff
@@ -1331,6 +1331,7 @@ You can add a `+` or `-` to display the list in ascending or descending order re
 - Query parameter can't be empty or filled with blank spaces.
 - Query parameter can accept a mix of all characters.
 - By default, the list appears in ascending order.
+- Query parameter `licenseType` sorts by two fields `licenseType` and `licenseStatus`.
```
Comment (madeline-brennan) [inline apiary.apib:1334]:
```suggestion
- Query parameter `licenseType` sorts by two fields: `licenseType` and `licenseStatus`.
```

**Review by madeline-brennan — CHANGES_REQUESTED** [review]
Just a small tweak

**Review by tkrisztian95 — APPROVED** [review]
Looks good. Thanks!

**Review by tkrisztian95 — APPROVED** [review]

**Review by madeline-brennan — APPROVED** [review]

**Review by george-haddad — APPROVED** [review]
:+1:

**Pull request merged**